extensions/amp-analytics/0.1/vendors.js

```diff
@@ -297,5 +297,28 @@ export const ANALYTICS_CONFIG = {
       'pageview': 'https://${host}${requestPath}?${basePrefix}',
       'click': 'https://${host}${requestPath}?${basePrefix}&pe=lnk_${linkType}&pev1=${linkUrl}&pev2=${linkName}',
     }
+  },
+
+  'infonline': {
+    'vars': {
+      "sv":"ke",
+      "ap":"1",
+    },
+    'transport': {'beacon': false, 'xhrpost': false, 'image': true},
+    'requests': {
+      'pageview': '${url}?st=${st}' +
+        '&sv=${sv}' +
+        '&ap=${ap}' +
+        '&co=${co}' +
+        '&cp=${cp}'
+    },
+    'triggers': {
+      'pageview': {
+        'on': 'visible',
+        'request': 'pageview',
+        'iframePing': true
```

Comment:
Change this to
'iframe' + /* TEMPORARY EXCEPTION */ + 'Ping'
to pass presubmit